webapp/src/js/services/message-contacts.js

```diff
@@ -10,51 +8,62 @@ angular.module('inboxServices').factory('MessageContacts',
     'use strict';
     'ngInject';
 
-    const listParams = function() {
-      return {
-        group_level: 1
-      };
-    };
+    const listParams = () => ({
+      group_level: 1
+    });
 
-    const conversationParams = function(id, skip) {
-      return {
-        reduce: false,
-        descending: true,
-        include_docs: true,
-        skip: skip,
-        limit: 50,
-        startkey: [ id, {} ],
-        endkey: [ id ]
-      };
-    };
+    const defaultLimit = 50;
```

Comment:
I think this would make more sense if it was called `minLimit`?

Reply:
done.